config.go

```diff
@@ -137,9 +152,23 @@ func newConfig(options ...Option) config {
 	return cfg
 }
 
-func withDBStatement(cfg config, query string) []attribute.KeyValue {
+func withDBStatement(ctx context.Context, cfg config, query string, args []driver.NamedValue) []attribute.KeyValue {
 	if cfg.SpanOptions.DisableQuery {
 		return cfg.Attributes
 	}
-	return append(cfg.Attributes, semconv.DBStatementKey.String(query))
+	attrs := append(cfg.Attributes, semconv.DBStatementKey.String(query))
+	if cfg.SpanOptions.ArgumentOptions.EnableAttributes {
+		opts := cfg.SpanOptions.ArgumentOptions
+		list := namedToInterface(args)
+		if opts.Skip == nil || (opts.Skip != nil && !opts.Skip(ctx, query, list)) {
```

Comment:
should we invert this if ? It's a little nicer to read
```go
// skip adding of db attributes if true
if opts.Skip != nil && opts.Skip(ctx, query, list) {
  	attrs = append(attrs, attribute.Bool("db.args.skipped", true))
} else {
        // add all the attributes
}
```

Reply:
Good point, that's nicer to read this way, thanks!